src/plugins.c

```diff
@@ -858,18 +1098,19 @@ load_plugins_from_path(const gchar *path)
 	GSList *list, *item;
 	gchar *fname, *tmp;
```

Comment:
while at it, moving the declaration of `fname` in the inner scope would be perfect :)